<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-01T11:21:29 IST" name="Default suite" finished-at="2025-04-01T11:26:17 IST" duration-ms="287826">
    <groups>
    </groups>
    <test started-at="2025-04-01T11:21:29 IST" name="Default test" finished-at="2025-04-01T11:26:17 IST" duration-ms="287826">
      <class name="ZapTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:ZapTest@a514af7]" started-at="2025-04-01T11:21:29 IST" name="setUp" finished-at="2025-04-01T11:21:34 IST" duration-ms="5269" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testActiveScan()[pri:0, instance:ZapTest@a514af7]" started-at="2025-04-01T11:21:34 IST" name="testActiveScan" finished-at="2025-04-01T11:21:35 IST" duration-ms="630" status="FAIL">
          <exception class="org.zaproxy.clientapi.core.ClientApiException">
            <message>
              <![CDATA[The external data provided is not valid.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.zaproxy.clientapi.core.ClientApiException: The external data provided is not valid.
at org.zaproxy.clientapi.core.ApiResponseFactory.getResponse(ApiResponseFactory.java:50)
at org.zaproxy.clientapi.core.ClientApi.callApi(ClientApi.java:392)
at org.zaproxy.clientapi.core.ClientApi.callApi(ClientApi.java:381)
at org.zaproxy.clientapi.gen.Context.importContext(Context.java:186)
at Util.ZapUtil.getContextAfterImporting(ZapUtil.java:96)
at Util.ZapUtil.performActiveScan(ZapUtil.java:79)
at ZapTest.testActiveScan(ZapTest.java:41)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- org.zaproxy.clientapi.core.ClientApiException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testActiveScan -->
        <test-method is-config="true" signature="tearDown(java.lang.reflect.Method)[pri:0, instance:ZapTest@a514af7]" started-at="2025-04-01T11:21:35 IST" name="tearDown" finished-at="2025-04-01T11:21:40 IST" duration-ms="4894" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void ZapTest.testActiveScan() throws org.zaproxy.clientapi.core.ClientApiException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:ZapTest@a514af7]" started-at="2025-04-01T11:21:40 IST" name="setUp" finished-at="2025-04-01T11:21:41 IST" duration-ms="1177" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testPassiveScan()[pri:0, instance:ZapTest@a514af7]" started-at="2025-04-01T11:21:41 IST" name="testPassiveScan" finished-at="2025-04-01T11:21:58 IST" duration-ms="16508" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPassiveScan -->
        <test-method is-config="true" signature="tearDown(java.lang.reflect.Method)[pri:0, instance:ZapTest@a514af7]" started-at="2025-04-01T11:21:58 IST" name="tearDown" finished-at="2025-04-01T11:22:01 IST" duration-ms="3071" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void ZapTest.testPassiveScan()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:ZapTest@a514af7]" started-at="2025-04-01T11:22:01 IST" name="setUp" finished-at="2025-04-01T11:22:02 IST" duration-ms="1133" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testSpider()[pri:0, instance:ZapTest@a514af7]" started-at="2025-04-01T11:22:02 IST" name="testSpider" finished-at="2025-04-01T11:26:13 IST" duration-ms="250920" status="FAIL">
          <exception class="org.zaproxy.clientapi.core.ClientApiException">
            <message>
              <![CDATA[The external data provided is not valid.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.zaproxy.clientapi.core.ClientApiException: The external data provided is not valid.
at org.zaproxy.clientapi.core.ApiResponseFactory.getResponse(ApiResponseFactory.java:50)
at org.zaproxy.clientapi.core.ClientApi.callApi(ClientApi.java:392)
at org.zaproxy.clientapi.core.ClientApi.callApi(ClientApi.java:381)
at org.zaproxy.clientapi.gen.Context.importContext(Context.java:186)
at Util.ZapUtil.getContextAfterImporting(ZapUtil.java:96)
at Util.ZapUtil.performActiveScan(ZapUtil.java:79)
at Util.ZapUtil.performSpidering(ZapUtil.java:69)
at ZapTest.testSpider(ZapTest.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- org.zaproxy.clientapi.core.ClientApiException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSpider -->
        <test-method is-config="true" signature="tearDown(java.lang.reflect.Method)[pri:0, instance:ZapTest@a514af7]" started-at="2025-04-01T11:26:13 IST" name="tearDown" finished-at="2025-04-01T11:26:17 IST" duration-ms="4135" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void ZapTest.testSpider() throws org.zaproxy.clientapi.core.ClientApiException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- ZapTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
